src/app/tests/suites/certification/Test_TC_CDOCONC_2_1.yaml

```diff
@@ -0,0 +1,132 @@
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: 145.1.2. [TC-CDOCONC-2.1] Attributes with server as DUT
+
+PICS:
+    - CDOCONC.S
+
+config:
+    nodeId: 0x12344321
+    cluster: "Carbon Dioxide Concentration Measurement"
+    endpoint: 1
+
+tests:
+    - label: "Wait for the commissioned device to be retrieved"
+      cluster: "DelayCommands"
+      command: "WaitForCommissionee"
+      arguments:
+          values:
+              - name: "nodeId"
+                value: nodeId
+
+    - label: "TH reads from the DUT the MinMeasuredValue attribute."
+      PICS: CDOCONC.S.A0001
+      command: "readAttribute"
+      attribute: "MinMeasuredValue"
+      response:
+          saveAs: MinMeasuredValue
+          constraints:
+              type: single
+              minValue: 0
+
+    - label: "TH reads from the DUT the MaxMeasuredValue attribute."
+      PICS: CDOCONC.S.A0002
+      command: "readAttribute"
+      attribute: "MaxMeasuredValue"
+      response:
+          saveAs: MaxMeasuredValue
+          constraints:
+              type: single
+              minValue: MinMeasuredValue
+
+    - label: "TH reads from the DUT the MeasuredValue attribute."
+      PICS: CDOCONC.S.A0000
+      command: "readAttribute"
+      attribute: "MeasuredValue"
+      response:
+          constraints:
+              type: single
+              minValue: MinMeasuredValue
+              maxValue: MaxMeasuredValue
+
+    - label: "TH reads from the DUT the PeakMeasuredValue attribute."
+      PICS: CDOCONC.S.A0003
+      command: "readAttribute"
+      attribute: "PeakMeasuredValue"
+      response:
+          constraints:
+              type: single
+              minValue: MinMeasuredValue
+              maxValue: MaxMeasuredValue
+
+    - label: "TH reads from the DUT the PeakMeasuredValueWindow attribute."
+      PICS: CDOCONC.S.A0004
+      command: "readAttribute"
+      attribute: "PeakMeasuredValueWindow"
+      response:
+          constraints:
+              type: elapsed_s
+              minValue: 0
+              maxValue: 259200
+
+    - label: "TH reads from the DUT the AverageMeasuredValue attribute."
+      PICS: CDOCONC.S.A0005
+      command: "readAttribute"
+      attribute: "AverageMeasuredValue"
+      response:
+          constraints:
+              type: single
+              minValue: MinMeasuredValue
+              maxValue: MaxMeasuredValue
+
+    - label: "TH reads from the DUT the AverageMeasuredValueWindow attribute."
+      PICS: CDOCONC.S.A0006
+      command: "readAttribute"
+      attribute: "AverageMeasuredValueWindow"
+      response:
+          constraints:
+              type: elapsed_s
+              minValue: 0
+              maxValue: 259200
```

Comment:
shouldn't  `604800`  be used? looking at PR https://github.com/CHIP-Specifications/connectedhomeip-spec/pull/7176/files

Reply:
Yeah @tobiasgraf - I'll make that change in another PR once that is merged to spec as I have to modify the XML too for that and trying to keep this PR clean (ish)